RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: patch
+
+This release adds support for `Django 2
+<https://www.djangoproject.com/weblog/2017/dec/02/django-20-released/>`_ in
+the hypothesis-django extra.
+
+This release drops support for Django 1.10, as it is no longer supported by
+the Django team.
```

Comment:
It might be worth being more explicit that e.g. it will probably continue to work for a while despite being unsupported, but that we will not accept patches to continue this state of affairs.

Reply:
The version pins will prevent you from installing Hypothesis with unsupported versions of Django (assuming you have a newish version of setuptools and pip), so I think I’ll leave this as-is.